src/Umbraco.Cms.Integrations.Automation.Zapier/App_Plugins/UmbracoCms.Integrations/Automation/Zapier/lang/en-US.xml

```diff
@@ -2,5 +2,7 @@
 <language>
 	<area alias="dashboardTabs">
 		<key alias="zapierDashboard">Zapier Integrations</key>
+		<key alias="promptDeleteTitle">Delete Registered Webhook</key>
```

Comment:
Possibly should use something here to be sure we don't clash with another package.  I.e. either rename the area to be `zapierDashboard` or rename the key aliases to be prefixed with `zapier...`.

Reply:
Updated alias and keys.